src/firefighter/slack/tasks/send_reminders.py

```diff
@@ -116,3 +126,81 @@ def send_reminders() -> None:
                 )
             )
     group(list_fn).apply_async()
+
+
+@shared_task(name="slack.send_reminders_to_commander")
+def send_reminders_to_commander() -> None:
```

Comment:
`send_reminders_to_commander` is misleading, it can also send reminders in the incident conversation. `send_postmortem_late_reminder`, or something else, but coherent with the message ID and name.

Reply:
Also some documentation of the business logic for this function (we send a message to the commander after XXX, if the condition still applies, we send to the incident conversation after XX, etc...)